ui/tests/integration/components/box-radio-test.js

```diff
@@ -1,23 +1,25 @@
 import { module, test } from 'qunit';
 import { setupRenderingTest } from 'ember-qunit';
 import sinon from 'sinon';
-import { render, click } from '@ember/test-helpers';
+import { render, click, settled } from '@ember/test-helpers';
 import hbs from 'htmlbars-inline-precompile';
 
 module('Integration | Component | box-radio', function(hooks) {
   setupRenderingTest(hooks);
 
   hooks.beforeEach(function() {
-    this.set('type', 'aws');
-    this.set('displayName', 'An Option');
-    this.set('mountType', '');
-    this.set('disabled', false);
+    this.type = 'aws';
+    this.displayName = 'An Option';
+    this.mountType = '';
+    this.disabled = false;
   });
 
+  const spy = sinon.spy();
```

Comment:
Pretty sure this needs to live within the `test` fn. But i know you're still working on this :)